comptests/prov/test_prov_k8s_cortx_cold_upgrade.py

```diff
@@ -75,58 +74,59 @@ def test_37357(self):
         remote_sol_path = self.prov_deploy_cfg["git_remote_path"] + \
-            "solution.yaml"
+                          "solution.yaml"
         solution_path = self.master_node_obj.copy_file_to_local(remote_path=remote_sol_path,
                                                                 local_path=self.local_sol_path)
         assert_utils.assert_true(solution_path[0], solution_path[1])
         image_dict = {"all_image": self.cortx_all_image,
                       "rgw_image": self.cortx_rgw_image}
-        local_path = self.deploy_lc_obj.update_sol_with_image(
+        local_path = self.deploy_obj.update_sol_with_image(
             self.local_sol_path, image_dict)
         assert_utils.assert_true(local_path[0], local_path[1])
         for node_obj in self.host_list:
-            resp = self.deploy_lc_obj.copy_sol_file(node_obj, local_sol_path=local_path[1],
-                                                    remote_code_path=self.
-                                                    prov_deploy_cfg["git_remote_path"])
+            resp = self.deploy_obj.copy_sol_file(node_obj, local_sol_path=local_path[1],
+                                                 remote_code_path=self.
+                                                 prov_deploy_cfg["git_remote_path"])
             assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 4: Done.")
 
         LOGGER.info("Step 5: Pull the images for upgrade.")
         for node_obj in self.host_list:
             for image in image_dict:
-                resp = self.deploy_lc_obj.pull_image(
+                resp = self.deploy_obj.pull_image(
                     node_obj, image_dict[image])
                 assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 5: Done.")
 
         LOGGER.info("Step 6: Start upgrade.")
         LOGGER.info("Upgrading CORTX image to version: %s.",
                     self.cortx_all_image)
-        resp = self.deploy_lc_obj.upgrade_software(self.master_node_obj,
-                                                   self.prov_deploy_cfg["git_remote_path"],
-                                                   upgrade_type="cold")
+        resp = self.upgrade_obj.upgrade_software(self.master_node_obj,
+                                                 self.prov_deploy_cfg["git_remote_path"],
+                                                 upgrade_type="cold")
         assert_utils.assert_true(resp[0], resp[1])
         LOGGER.info("Step 6: Done.")
 
-        LOGGER.info(
-            "Step 7: Check if installed version is equals to installing version.")
-        installed_version = self.deploy_lc_obj.generate_and_compare_version(self.cortx_all_image, installed_version)
+        LOGGER.info("Step 7: Check if installed version is equals to installing version")
+        installed_version = self.deploy_obj.generate_and_compare_both_version(
+            self.cortx_all_image, installed_version)
```

Comment:
Does this method check for equal version ? I guess not.

Reply:
No